src/touched.ts

```diff
@@ -0,0 +1,50 @@
+import Board from './board';
```

Comment:
Ten plik nie powinien istnieć, proszę skoryguj to

Reply:
Racja, sorki, przeoczyłem to :)